### ISSUE TITLE
LAKES missing lake information

### ISSUE THREAD

**Issue by katiesaunders** [MFEh2o/db#14]
AL, CW, LN, LT, NN, OX, RT, TH, EE, EW are missing location information. Many others need lat/long information.

**Comment by kaijagahm** [comment]
Possibly address at the same time as #26

**Comment by kaijagahm** [comment]
Obtained information for EE, EW, TH, and RT from newLakeIDs_zieglerYOY.csv. Still need info for the others.

**Comment by kaijagahm** [comment]
Created map for unknown lakes: https://www.google.com/maps/d/drive?state=%7B%22ids%22%3A%5B%221ThoMV5nAz8IHaosP5wLiqLfOHi0Lw_Je%22%5D%2C%22action%22%3A%22open%22%2C%22userId%22%3A%22110377163588285425288%22%7D&usp=sharing. Best guesses for where they might be.

**Comment by kaijagahm** [comment]
Received files from the Bluegill 2016 and Tres Amigos 2016 projects from Chris and Stuart. Checked these files: no mention of any of the 6 missing lakes, despite these projects having been associated with their updateID's.

**Comment by kaijagahm** [comment]
Pulled information for the 6 remaining lake ID's (AL, CW, LN, LT, NN, OX) from all database tables. They're all associates with projects 25 (Residence time - Respiration 2015) and 37 (FishScapes Survey).

**Comment by kaijagahm** [comment]
Wrote to Alex Ross to ask if he knows. We seem to be all set on LT (Lost) based on the google map--filled in geog. info. Have to ask Stuart whether we should use the DNR area/depth measurements, or whether those come from somewhere else. The rest of the lakes are from the Residence Time project, which we have to ask Stuart about.

**Comment by kaijagahm** [comment]
Received NHLR file from Stuart, as well as a paper from Patrick Kelly's undergrad that contains locations for OX and LN. LN is indeed Langford, not Longford––changed spelling. Removed CW because we decided it's a typo of CY. Began addressing missing/misrounded lat/long values. Used NHLD data to identify an additional candidate for Allen lake.

**Comment by joneslabND** [comment]
@kaijagahm seems like the best bet might be for you to contact Patrick Kelly directly. I'll e-introduce you now.

**Comment by kaijagahm** [comment]
Email to Patrick:
`I'm missing information (lat/long, surface area, max depth) on Allen Lake (AL), North North Crab Lake (NN [note: distinct from North Crab, which is NR]), and Little Oxford Lake (OX). Of those three, it looks like only OX is referenced in the residence time paper ("The Relationship Between Dissolved Organic Carbon Lability and Water Residence Time and its Impact on Bacterial Respiration"), which Stuart sent me, but AL and NN are also associated with that project in the database.`

Patrick:
`As for the lakes, just to clear up some confusion, I remember going to a North Crab Lake - which is the name of the lake, but I don’t remember a north North Crab Lake. Also, I think I maybe remember an Allen Lake but it obviously didn’t get included, so I don’t have any information on it. I Google Mapped an Allen Lake which is the lake just north of the field station, but I’m not sure I sampled that. I remember Langford Lake up there, but not an Allen Lake. The only other thing I can think of, is that I sampled a lake near the property where there was a land owner on it, and his name may have been Allen. But I think the lake was unnamed, so I doubt I would be able to piece together any information about it this long after. As for Little Oxford, is it possible that it’s Little Oxbow? Anyways, I have lake area and watershed area data (see attached) for Little Oxbow and North Crab if it’s useful, but we didn’t collect any data on max depth as far as I remember.`

So we have no idea what's going on with NN (North North Crab). Just delete?
What do I do about Allen lake? Not sure why he says "but it obviously didn't get included"--it's in the database, attached to a few POC, NUTRIENTS, DOC, and CHLOROPHYLL samples, all from projectID 25, which is Patrick's project. Looks like maybe he didn't sample it super extensively?
Yes, Little Oxford probably is Little Oxbow. Can use the data he sent.
